operator/controller/src/main/java/io/apicurio/registry/operator/resource/app/AppDeploymentResource.java

```diff
@@ -151,4 +163,16 @@ public static Container getContainerFromPodTemplateSpec(PodTemplateSpec pts, Str
         }
         return null;
     }
+
+    private static void configureAuth(Map<String, EnvVar> envVars, AppAuthSpec appAuthSpec) {
```

Comment:
Can you move this to a separate class in the `feat` package? It'll be easier to find implementation of each feature.

Reply:
+1